packages/@vuepress/theme-default/src/client/components/ToggleDarkModeButton.vue

```diff
@@ -42,10 +42,16 @@
 </template>
 
 <script setup lang="ts">
-import { useDarkMode } from '../composables'
+import { computed } from 'vue'
+import { useDarkMode, useThemeLocaleData } from '../composables'
 
 const isDarkMode = useDarkMode()
 const toggleDarkMode = (): void => {
   isDarkMode.value = !isDarkMode.value
 }
+
+const title = computed(() => {
+  const themeLocale = useThemeLocaleData()
```

Comment:
Put the `use` function outside computed

Reply:
In fact, we may not need a computed value here. We should set the default value in `assignDefaultLocaleOptions`